tests/test_keywords.py

```diff
@@ -403,6 +403,18 @@ def test_initial_stress_shell(ref_string):
     assert stress4.hisv[5] == 0.163
 
 
+@pytest.mark.keywords
+def test_initial_temperature(ref_string):
+    tin = kwd.InitialTemperatureNode()
+    tin.loads(ref_string.test_initial_temperature_node_string)
+    assert tin.write() == ref_string.test_initial_temperature_node_string
+    tis = kwd.InitialTemperatureSet()
+    tis.loads(ref_string.test_initial_temperature_set_string)
+    print(tis.write())
```

Comment:
@RaphaelHeiniger could you please submit a PR that removes the print statements?